### DISCUSSION TITLE
Icon Request: fa-jump-over (fa-skip)

### DISCUSSION THREAD

**Discussion by AndreVianna** [FortAwesome/Font-Awesome#21169] [Icon Requests]
It is a very simple icon that I can't find in any library.
It should looks like a arrow going over a dot. 
It is used in workflows or wizards to indicate that you can skip a step, or avid some part of a task.
It is not equivalent to the request  #3575!
The icon you mention as duplicate in #3575 is "jump to".
What I need is something like "jump over"

In a sequence of steps (in a Wizard or Workflow) you may be able to skip one step or task.
The icon should have the arrow going around the dot. not end on it. To indicate that something was skipped or avoided.

Like this (sorry I don't have a good design tool):

```
  ___
-/ o \->
```

**Comment by pixsrv** [comment]
+1

![image](https://cloud.githubusercontent.com/assets/10224591/6265565/3d60037c-b830-11e4-9708-9e163aa0ddab.png)
(taken from chrome)

**Comment by AndreVianna** [comment]
Yes... That is precisely what I mean.
Thanks

**Comment by drickstan** [comment]
+1

**Comment by justi** [comment]
+1

**Comment by ttristan** [comment]
+1

**Comment by hanishassim** [comment]
+1

**Comment by Ali-00-Ridha** [comment]
+1

**Comment by rganz** [comment]
+1

**Comment by vishalknitian** [comment]
+1

**Comment by tagliala** [comment]
@AndreVianna apologies for reopening this old request

Since there is still request for this icon, I would like to reopen to preserve the old upvotes and avoid to start from scratch.

If you don't mind, please unsubscribe so you will not receive notification.

Feel free to close again this issue and I will lock it to force opening a new one

**Comment by steyo** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
